demos/object_detection_demo/jupyter-python/requirements.txt

```diff
@@ -0,0 +1,10 @@
+requests
+jupyterlab
+ipywidgets
+voila
+matplotlib
+openvino
+opencv-python-headless
+pyyaml
+ipython==7.10
+numpy==1.19
```

Comment:
ipython gives an annoying irrelevant warning message in the notebook with their most recent version. 7.10 does not give that warning. With numpy I ran into issues because OpenVINO installed I think 1.16 and that did not work with another package. I read that there would be standardization of numpy across OpenVINO tools though. If `pip install openvino` will install numpy 1.18 or 1.19 I think I can avoid this version pin.

Reply:
> ipython gives an annoying irrelevant warning message in the notebook with their most recent version. 7.10 does not give that warning.

This doesn't seem like a good enough reason. Generally, any pinned package is a liability in that it may stop working on a newer version of Python, be incompatible with the user's other installed packages or contain security vulnerabilities. A warning is comparatively harmless.

> With numpy I ran into issues because OpenVINO installed I think 1.16 and that did not work with another package.

I don't see why it would do that. Looking at the wheel, it requires `numpy (>=1.16.3)`, so it would install the latest version by default. Moreover, if it really does require 1.16, then pinning it to 1.19 would just create an inconsistent environment, which isn't ideal.

Reply:
After installing openvino with `pip install openvino`, if you install the OMZ requirements-pytorch.in, to be able to convert PyTorch models with Model Converter, it installs onnx. This requires a higher Numpy version than 1.16.3, and pip installs Numpy 20 then. See screenshot. 
![image](https://user-images.githubusercontent.com/77325899/110916414-0ff3f780-8319-11eb-8747-98b7ab1e0378.png)

It is my understanding that Numpy versions would be standardized across OpenVINO tools to either 1.18 or 1.19. If that is the case then I can remove this pinning. @mryzhov will the final version of the PyPI packages have a different numpy version requirement?

If Numpy 20 doesn't cause problems I can also remove the version pinning.

Reply:
We noticed when people tested this that they were a bit alarmed by warnings that were not related to OpenVINO, like this one. Inexperienced devs will not recognize it as a harmless warning. I'll remove the pinning for this notebook though. I hope that ipython will fix this in a future version.

Reply:
> After installing openvino with pip install openvino, if you install the OMZ requirements-pytorch.in, to be able to convert PyTorch models with Model Converter, it installs onnx. This requires a higher Numpy version than 1.16.3, and pip installs Numpy 20 then.

What I don't understand is why 1.16.3 was installed to begin with. As I've said, `openvino` only has a lower bound on the version of NumPy (at least in the last release).

Reply:
I'll delete the numpy version pinning from the notebook. The `openvino-dev` pip package installed 1.16.3 (see screenshot above). The latest testing wheel (2774) installs 1.16.6 and pins the version to this. There's still a conflict error message from pip though, if you `pip install -r requirements-tensorflow.in` (in model downloader directory) it installs numpy 1.19.5.

```
pkginfo -f requires_dist openvino_dev-2021.3.0-2774-py3-none-any.whl
requires_dist: ['defusedxml (>=0.5.0)', 'scipy (==1.5.4)', 'jstyleson (==0.0.2)', 'numpy (==1.16.6)', 'addict (==2.2.1)', 'pandas (==0.24.2)', 'hyperopt (==0.1.2)', 'networkx (==2.2)', 'tqdm (==4.31.1)', 'texttable (==1.6.3)', 'py-cpuinfo (!=5.0,!=6.0)', 'PyYAML (>=5.4.1)', 'pillow (>=8.1.0)', 'scikit-image', 'scikit-learn', 'yamlloader', 'shapely', 'nibabel', 'pydicom', 'sentencepiece', 'tokenizers', 'editdistance', 'parasail', 'fast-ctc-decode', 'rawpy', 'nltk', 'opencv-python', 'openvino (==2021.3.0)']`
``